### PR TITLE
PR for Release 1.4

### PR DESCRIPTION
**Pull request by apeltzer**
Bump this to V1.4! 

For details check out the CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # nf-core/tools: Changelog
 
+## [v1.4](https://github.com/nf-core/tools/releases/tag/1.4) - 2018-12-12
```
Comment (sven1103) [inline CHANGELOG.md]:
```suggestion
## [v1.4](https://github.com/nf-core/tools/releases/tag/1.4) - 2018-12-12 Tantalum Butterfly
```

**Comment by ewels** [comment]
This pull request **fixes 11 alerts** when merging 9219e2b90143c3d52f4abed66b6d0cad02b53903 into aa947db2782715433fe1b9216de691e942370165 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-2748a38a8aa19e8a1e118d77d689600c03fe1cc2)

**fixed alerts:**

* 11 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/installation.md [hunk 2]
```diff
@@ -100,8 +100,15 @@ This is slower and less reproducible than the above, but is still better than ha
...
-## Appendices
+See [`docs/configuration/adding_your_own.md`](configuration/adding_your_own.md)
+
+## 4) Reference genomes
+
+See [`docs/configuration/reference_genomes.md`](configuration/reference_genomes.md)
+
+## 5) Appendices
 
 #### Running on UPPMAX
```
Comment (ewels) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/installation.md]: We should probably remove this now...

**Comment by ewels** [comment]
This pull request **fixes 11 alerts** when merging 06507cea4088a19acf55d694fa6af27489384a08 into aa947db2782715433fe1b9216de691e942370165 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-c2acd57bf67c14a1496d6a962a96cc232510f53d)

**fixed alerts:**

* 11 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**